### PR TITLE
export url

### PR DESCRIPTION
**Pull request by michaelglass**
the example usage references a non-existant export

this PR
- exports the page URL
- and uses it in the readme example usage

(as well as some minor cleanups)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +57,7 @@ jobs:
 | name         | default | options                                           |
 | ------------ | ------- | ------------------------------------------------- |
 | output-dir   | doc     | Your YARD product dir                             |
-| ruby-version | 3.2     | See options in https://github.com/ruby/setup-ruby |
+| ruby-version | 3.2     | See options in <https://github.com/ruby/setup-ruby> |
```
Comment (kachick) [inline README.md:60]: Is `<>` correct escape in GFM?

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links
https://www.markdownguide.org/basic-syntax/

In my understanding, `[]()` is  the ideal way in Markdown. However GitHub markups current styles

File: action.yml [hunk 2]
```diff
@@ -13,6 +13,11 @@ inputs:
     description: 'YARD runner will be executed with this ruby version'
     required: true
     default: '3.2'
+outputs:
+  page_url:
+    description: "github page url"
+    value: ${{ steps.deployment.outputs.page_url }}
```
Comment (kachick) [inline action.yml:19]: Which behaviors will use this one? I know actions/deploy-pages@v1 outputs as this. However personally I don't use this...

File: action.yml [hunk 3]
```diff
@@ -27,7 +32,7 @@ runs:
     - uses: actions/configure-pages@v2
     - uses: actions/upload-pages-artifact@v1
       with:
-        # Upload entire repository
+        # By default, upload the docs directory
```
Comment (kachick) [inline action.yml:35]: Nice catch 👍

File: README.md [hunk 4]
```diff
@@ -36,13 +36,15 @@ concurrency:
...
   deploy_yard:
+    # the deploy environment (not to be confused with env)
     environment:
       name: github-pages
       url: ${{ steps.deployment.outputs.page_url }}
     runs-on: ubuntu-latest
     name: Build and deploy YARD
     steps:
       - uses: kachick/deploy-yard-to-pages@v1.2.0
+        id: deployment
```
Comment (kachick) [inline README.md:47]: I think this might make confusion with actions/deploy-pages@v1

And I want to keep minimum for required config.

**Review by kachick — COMMENTED** [review]
Thank for your PR!

Perfectly agreed, this action is needless to be used.👍 for https://github.com/rwx-research/rspec-abq/pull/88
(I just wanted to have single repository to manege YARD deploying 
 knowledges. However the abstraction cost looks higher than the benefit)

File: action.yml (hunk 3 above)
Comment (kachick) [reply action.yml:35]: cherry-picked as https://github.com/kachick/deploy-yard-to-pages/commit/c6bddace4c6bf991e320e8694c31dc4ff1a5c231

File: README.md (hunk 1 above)
Comment (michaelglass) [reply README.md:60]: I got it from markdownlint, https://github.com/DavidAnson/markdownlint/blob/main/doc/md034.md

File: README.md (hunk 4 above)
Comment (michaelglass) [reply README.md:47]: without this change, the `enviroment` field above has an undefined URL and there is no `view deployment` button on PRs / on the deployments page

CleanShot 2023-01-04 at 15 28 19@2x

an alternative to this PR: I'd suggest removing the `url` from the `environment` field above.

File: action.yml (hunk 2 above)
Comment (michaelglass) [reply action.yml:19]: [the deployment environment](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment)

**Comment by michaelglass** [comment]
fwiw having the github action in the first place inspired us to do much more documentation! I think it's nice to have around even if folks eventually inline it!

File: README.md (hunk 4 above)
Comment (kachick) [reply README.md:47]: Oh... I never know the button exists in pages feature. 😮 🙇‍♂️

**Review by kachick — APPROVED** [review]
Thanks for letting me know! This change looks reasonable 👍

File: README.md (hunk 1 above)
Comment (kachick) [reply README.md:60]: Thanks!

In this repository, markdown table will be formatted with dprint. It alerting for the indent https://github.com/kachick/deploy-yard-to-pages/actions/runs/3837091084/jobs/6536103406

I'll fix it in another commits.

Comment (kachick) [reply README.md:60]: https://github.com/kachick/deploy-yard-to-pages/commit/290fe8afbb89bfcb9784f297e6000cb8e942a92b

**Pull request merged**